lib_webserver/gen/makefs.py

```diff
@@ -1,3 +1,4 @@
+# Copyright (c) 2012-2015, XMOS Ltd, All rights reserved
```

Comment:
Hi @pthedinger, I suspect the source check needs running a second time now that this file has been modified in 2018!